mobipick_bringup/nodes/gripper_joint_state_publisher.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
In general I believe it's better to use `#!/usr/bin/env python3`. The difference is that this variant plays nicer with `venv`, because it uses the `python3` from your current `PATH`, not a hardcoded location.

Reply:
you're right!